tools/ProductConstructionService.ReproTool/Operations/Operation.cs

```diff
@@ -0,0 +1,224 @@
+﻿// Licensed to the .NET Foundation under one or more agreements.
+// The .NET Foundation licenses this file to you under the MIT license.
+
+using Maestro.Data;
+using Microsoft.DotNet.DarcLib.Helpers;
+using Microsoft.DotNet.DarcLib.VirtualMonoRepo;
+using Microsoft.DotNet.ProductConstructionService.Client;
+using Microsoft.DotNet.ProductConstructionService.Client.Models;
+using Microsoft.EntityFrameworkCore;
+using Microsoft.Extensions.Logging;
+using Octokit;
+
+namespace ProductConstructionService.ReproTool.Operations;
+internal abstract class Operation(
+    ILogger<Operation> logger,
+    GitHubClient ghClient,
+    BuildAssetRegistryContext context,
+    IProductConstructionServiceApi localPcsApi)
+{
+    protected const string MaestroAuthTestOrgName = "maestro-auth-test";
+    protected const string VmrForkRepoName = "dotnet";
+    protected const string VmrForkUri = $"https://github.com/{MaestroAuthTestOrgName}/{VmrForkRepoName}";
+    protected const string ProductRepoFormat = $"https://github.com/{MaestroAuthTestOrgName}/";
+    protected const long InstallationId = 289474;
```

Comment:
Is this really needed? The first subscription you add should look it up correctly.

Reply:
not anymore I guess, we used to need it tho